### DISCUSSION TITLE
"There is no allowed FLRW universe here."

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#193] [Cosmology Models]
[https://tritonstation.com/2023/12/20/holiday-concordance/](https://tritonstation.com/2023/12/20/holiday-concordance/)

> We have painted ourselves into a corner. The usual interpretation is that we have painted ourselves into the correct corner: we live in this strange LCDM universe. It is also possible that there really is nothing left, the concordance window is closed, and we’ve falsified FLRW cosmology.

> Does the whole picture hang together so well that it must be correct? Or can it be falsified?

**Comment by mikehelland** [comment]
I'm curious as to what some of you may think of the following claims:

* there is no connection between cosmological phenomena (ie, redshifts, time dilation, CMB) and gravity 

The fundamental assumption Einstein made back in 1917 is that gravity has some effect on the cosmos at infinite scales.

This assumption has never been explicitly tested. 

We have to consider **the null hypothesis**, specifically, there is zero correlation between how much matter and the strength of gravity in the universe, and the observed redshifts.

I think a valid interpretation of the _supernovae cosmology project_ could be, not as evidence for dark energy counter-acting the effects of gravity on the universe, but as evidence that gravity doesn't affect the universe on those scales at all. 

Off hand, do you know about how many alternative models fall into the category of "the null hypothesis", with respect to redshifts and gravity? Or are all they gravity "informed" some how?

**Reply by HanDeBruijn** [reply]
@mikehelland 

>- there is no connection between cosmological phenomena (ie, redshifts, time dilation, CMB) and gravity

With the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs) there IS a connection via _inertial **mass** = gravitational **mass**_.

- Cosmological redshift is intrinsic. Therefore it is caused by (more aged, hence) lower electron/proton mass. The [Rydberg constant](https://en.wikipedia.org/wiki/Rydberg_constant#Value) with lower masses $m_e$ (and $m_p$) corresponds with the wavelength $\lambda$ of any emitted radiation being longer in the past.
- The frequency $\nu$ of atomic clocks is likewise slowed down by lower mass $m_e$ (and $m_p$), because wavelength and frequency are coupled with $\nu = c_0/\lambda$ where lightspeed $c_0$ is assumed constant. There is a [_Time Dilated Past_](https://github.com/orgs/a-cosmology-group/discussions/160).
- According to theory in [Let there be CMB](http://www.alternatievewiskunde.nl/cosmology/CMBR.htm) the Cosmic Microwave Background Radiation is caused by (the self) gravity (field). But I'm not so sure (anymore), because of findings in [thread 189](https://github.com/orgs/a-cosmology-group/discussions/189#discussioncomment-7663302) and theories by other people here.

>there is zero correlation between how much matter and the strength of gravity in the universe, and the observed redshifts.

- Concerning the first "and": it is NO question that Variable Mass means Variable Gravity. Some people even [think](http://www.alternatievewiskunde.nl/cosmology/Ghosh.htm) that gravity itself is redshifted. Presence of the [Standard gravitational parameter](https://en.wikipedia.org/wiki/Standard_gravitational_parameter) - defined as the product $GM$ (with $M$ variable according to VMT) - does not exclude such an interpretation.
